luigi/scheduler.py

```diff
@@ -1487,6 +1494,55 @@ def _first_task_display_name(self, worker):
         else:
             return task_id
 
+    @rpc_method()
+    def jobs(self):
+        try:
+            if configuration.get_config().get("kubernetes", "auth_method") == "kubeconfig":
+                config.load_kube_config()
+            else:
+                config.load_incluster_config()
+
+            batch_api_v1 = client.BatchV1beta1Api()
+
+            namespace = configuration.get_config().get("kubernetes", "namespace", "default")
```

Comment:
For this part is there any possibility of using non default namespace? I think create a dynamic config can be helpful as well

Reply:
@irvifa maybe the answer is here https://github.com/spotify/luigi/pull/2923#discussion_r413960107? One can set the `namespace` cluster value in Luigi configurations and then it won't be the `default` value. Does this answer your comment?